tools/notarize

```diff
@@ -55,18 +56,24 @@ if [[ "${NOTARIZE}" == "1" ]]; then
   sudo mount -t hfsplus -o loop "${DMG}" /tmp/acorn_mount
   sudo cp -R "${DIR}"/* /tmp/acorn_mount
   sudo umount /tmp/acorn_mount
+  echo "Built ${DMG}!"
 
   # Notarize and staple the DMG
+  echo "Notarizing and stapling ${DMG}..."
```

Comment:
How would you feel about splitting this up? The idea being that we would be able to identify which command failed.

Reply:
Ah I see. Sounds like a plan.